tardis/model/base.py

```diff
@@ -84,6 +84,47 @@ def r_outer(self):
         return self.geometry.r_outer.values * self.geometry_units["r_outer"]
 
 
+class Radial1DGeometry:
+    """
+    Holds information about model geometry
+
+    Parameters
+    ----------
+    radius : astropy.units.quantity.Quantity
+    velocity : astropy.units.quantity.Quantity
+    """
+
+    def __init__(self, radius, velocity):
+        self.geometry_df = pd.DataFrame(
+            {"radius": radius, "velocity": velocity}
```

Comment:
Dataframes can't read astropy quantities, so they need to be explicitly set to CGS units with the `.to_cgs` method and then converted to a pure numpy array with the `.value` method.